File: populate_bcdb.js
const driver = require('bigchaindb-driver')
const bip39 = require('bip39');
const bdb_root_url= 'http://localhost:9984/api/v1/'
const alice = new driver.Ed25519Keypair(bip39.mnemonicToSeed("Secret").slice(0, 32));
const bob = new driver.Ed25519Keypair()
const conn = new driver.Connection(bdb_root_url)

claims = [
 { 'data': { 'ipfs_file_hash': 'QmTtVrUR41AoSWYtbnow53X3fmyBMoKj7Gs6ekLQZ5e4Cf'}},
 { 'data': { 'ipfs_file_hash': 'QmPLDZM9kV9A1RdRuPVQHqpeok5paEx7k9S14STZfjZBfg'}},
 { 'data': { 'ipfs_file_hash': 'QmeA9r15SDjPtLuu5cpHT3YD6XPHrjiNZiMgzBieeoMAJ8'}},
 { 'data': { 'ipfs_file_hash': 'QmQGqFFauYC7Vi7axpxZrGXUzvKtPjpATj1Gy4395fBVye'}},
 { 'data': { 'ipfs_file_hash': "QmZoEfFdnXMRNnRfq7KTxuh3TnfoV3eTWNmGAQUyzgtmgq"}},
 { 'data': { 'ipfs_file_hash': "Qmbf3UgNJTQdnXGzAXyqh5QxoPQihQuxrV9JT7o9qsdLG3"}},
 { 'data': { 'ipfs_file_hash': "QmeS8QPsE3HJpGoN8Evmn2GwbNoqjqUyKXKwdPY7ZRPeaF"}}];



const metadata = {'notcreated': 'status'}
for(var i=0; i < claims.length; i++) {
	console.log(claims[i])
	const txCreateAliceSimple = driver.Transaction.makeCreateTransaction(
        claims[i],
        metadata,
        // A transaction needs an output
        [ driver.Transaction.makeOutput(
                        driver.Transaction.makeEd25519Condition(alice.publicKey))
        ],
        alice.publicKey
)
const txCreateAliceSimpleSigned = driver.Transaction.signTransaction(txCreateAliceSimple, alice.privateKey)
conn.postTransactionCommit(txCreateAliceSimpleSigned)

}

File: populate_bcdb.py
#!/usr/bin/python3
from bigchaindb_driver import BigchainDB
from bigchaindb_driver.crypto import generate_keypair


#tokens = {}
#tokens['app_id'] = 'YOUR_APP_ID'
#tokens['app_key'] = 'YOUR_APP_KEY'
bdb_root_url= 'http://localhost:9984/'
#bdb = BigchainDB('https://test.bigchaindb.com', headers=tokens)

bdb = BigchainDB(bdb_root_url)
alice = generate_keypair()
claim_1 = {'data': {'ipfs_file_hash': 'QmTtVrUR41AoSWYtbnow53X3fmyBMoKj7Gs6ekLQZ5e4Cf'},}
claim_2 = {'data': {'ipfs_file_hash': 'QmPLDZM9kV9A1RdRuPVQHqpeok5paEx7k9S14STZfjZBfg'},}
claim_3 = {'data': {'ipfs_file_hash': 'QmeA9r15SDjPtLuu5cpHT3YD6XPHrjiNZiMgzBieeoMAJ8'},}
claim_4 = {'data': {'ipfs_file_hash': 'QmQGqFFauYC7Vi7axpxZrGXUzvKtPjpATj1Gy4395fBVye'},}
claim_5 = {'data': {'ipfs_file_hash': 'QmZoEfFdnXMRNnRfq7KTxuh3TnfoV3eTWNmGAQUyzgtmgq'},}
claim_6 = {'data': {'ipfs_file_hash': 'QmeEMGyyX94Krwj6nqgtk891VfeoDk4aQassHRggWp2nKE'},}
claim_7 = {'data': {'ipfs_file_hash': 'QmQACrt9PGVwgG2g7Qttf4FaQcnEv6pAWtuP2ztYf569jZ'},}


claims = [claim_1, claim_2, claim_3, claim_4, claim_5, claim_6, claim_7]
print(alice)
metadata ={'bill-created' : 'no'}
for claim in claims:
	prepared_creation_tx = bdb.transactions.prepare(
		operation='CREATE',
		signers=alice.public_key,
		asset=claim,
		metadata=metadata
	)
	fulfilled_creation_tx = bdb.transactions.fulfill(prepared_creation_tx, private_keys=alice.private_key)
	print("Inserting Claim {}".format(claim));
	bdb.transactions.send_commit(fulfilled_creation_tx)
